plugins/modules/rma_workflow_manager.py

```diff
@@ -563,8 +615,13 @@ def device_exists(self, identifier, identifier_type):
                     device = response['response'][0]
                     device_id = device.get('id')
                     serial_number = device.get('serialNumber')
-                    if device_id and serial_number:
-                        return device_id, serial_number
+                    device_name = device.get('hostname')
+                    series_name = device.get('series')
+                    family_name = device.get('family')
+                    reachability_status = device.get('reachabilityStatus')
+                    platform_id = device.get('platformId')
+                    if device_id and serial_number and device_name and series_name and family_name and reachability_status and platform_id:
+                        return device_id, serial_number, device_name, series_name, family_name, reachability_status, platform_id
```

Comment:
in line number 602 and 633, we are returning two values but here, 7 values.. Will it not impact later in the caller?

Reply:
Updated the code accordingly.